setup.py

```diff
@@ -96,9 +90,24 @@ def run_setup(with_cython):
                                 extra_link_args=['-O2', '-march=core2', '-mtune=corei7'])
         else:
             compile_args = {}
-        ext_modules = [Extension("pykrige.lib.cok", ["pykrige/lib/cok.pyx"], **compile_args),
-                       Extension("pykrige.lib.lapack", ["pykrige/lib/lapack.pyx"], **compile_args),
-                       Extension("pykrige.lib.variogram_models", ["pykrige/lib/variogram_models.pyx"], **compile_args)]
+        # Transfered python 3 switch here. On python 3 machines, will use lapack_py3.pyx
+        # instead of lapack.pyx to build .lib.lapack
+        if sys.version_info[0] == 3:

```

Comment:
Yes, sorry, I think we can just remove [lines 19-20](https://github.com/bsmurphy/PyKrige/blob/master/.travis.yml#L20) from `.travis.yml` that shouldn't be critical/necessary IMO, as we a) download the latest miniconda verison b) update it once on line 18.